### DISCUSSION TITLE
Batteries no longer updating

### DISCUSSION THREAD

**Discussion by badnetmask** [andrew-codechimp/HA-Battery-Notes#1749] [General]
For whatever reason no batteries are updating at all. I've already enabled debug logging, forced a refresh, but didn't see any errors. I also deleted one of the devices, added it again, and it came back with the exact same battery level as before. What else can I try, other than removing the entire integration?

**Comment by andrew-codechimp** [comment]
Very strange, the Battery+ sensor does not store its value, it's just a listener on the original battery adding attributes and events.
Just tried on latest HA version with my fake battery and it tracks the original battery.
I'd expect an error on the log if there was a failure attaching the listener.
Are you looking at the device screen where the raw sensors are, just in case there's something odd with dashboard refresh?

**Reply by badnetmask** [reply]
Yes, I looked directly at the device, and both "Battery level" and "Battery+" are showing the same incorrect number.

**Reply by andrew-codechimp** [reply]
Ah, so Battery level is incorrect, that will be the real device or integration that is supplying that level rather than Battery Notes.  Battery+ just echo's that value from the real integration, it doesn't go to the original device for it.

**Reply by badnetmask** [reply]
Okay, that makes sense. I triggered a refresh on the device integration and it quickly updated the battery.
I'm sorry for not even looking there in the first place.
Thanks for the quick response!

**Reply by andrew-codechimp** [reply]
No problem, it hopefully explains to others how battery notes works.
